### ISSUE TITLE
Wingpanel Icon In Dock

### ISSUE THREAD

**Issue by swt83** [Lafydev/wingpanel-indicator-namarupa#2]
I installed this and it seems to work well.  My only complaint is my computer shows Wingpanel as an icon in the dock that cannot be closed and appears there permanently.

**Comment by Lafydev** [comment]
yes, it come's after a click on the settings button, it's supposed to open a window with a lot of configurations, I'm not sure if I've time for this, but I want to stay compatible with donadigo. 
Thank you, for your tests. :-)

**Comment by swt83** [comment]
When I clicked on settings it didn't show me any options.  It just told me how many icons it was holding.

**Comment by Lafydev** [comment]
Yes, you're right, it's not useful.
I'm not sure what to do ? If donadigo want to take my PR, here is what it was supposed to do : 
 https://github.com/donadigo/wingpanel-indicator-namarupa/issues/1
Or have another fork without settings? 
I keep this issue open for the moment.

**Comment by Lafydev** [comment]
**Question to any user of namarupa**  : what would be needed for you in settings ?

**Comment by Eldhrimer** [comment]
> Question to any user of namarupa : what would be needed for you in settings ?

I'd like to be able to deactivate some icons that have truly not purpose of being in the systray (like steam or zoom) to help declutter the pop up. 
I would be nice to have a setting to allow the pop up to not be dismissed when I click somewhere else, sometimes I need to be able to monitor some sensor while doing some specific work. I think a setting for this could be pretty cumbersome way to implement, maybe a toggle on the pop up is a better compromise, but I don't know if it is feasible.

**Comment by MvBonin** [comment]
Maybe we could even combine wingpanel-indicator-ayatana and wingpanel-indicator-namarupa.
Settings could hold options for hiding indicators, showing them on wingpanel directly or showing them in ... namarupa menu.
This would declutter everything but give the user options to choose, what they want to see where.

For example, you could show Nordvpn indicator in wingpanel directly and Steam, Nextcloud, etc in the ... menu of namarupa.
